manim/mobject/opengl/opengl_mobject.py

```diff
@@ -93,26 +116,26 @@ class OpenGLMobject:
 
     def __init__(
         self,
-        color=WHITE,
-        opacity=1,
-        dim=3,  # TODO, get rid of this
+        color: ParsableManimColor | list[ParsableManimColor] = WHITE,
```

Comment:
```suggestion
        color: ParsableManimColor | Sequence[ParsableManimColor] = WHITE,
```
Technically, this should even be an `Iterable` (after looking at `ManimColor.parse`, the extra stuff given by `Sequence` is unneeded), but in the interest of keeping it consistant with the rest of the codebase I have suggested `Sequence`

Reply:
About that, I've seen other sections in the code which use `Iterable` (in fact, I could have sworn that `VMobject.__init__()` did the same, but it used a `list`)... so I ended up using `Iterable` as it's the most fitting option IMO.